### PR TITLE
fix: regex for matching capture-area.geojson

### PR DESCRIPTION
**Pull request by amfage**
#### Motivation

The previous regex looked for a match for the start of the filename i.e. `^capture-area.geojson$`, however, when running on Argo Workflows, the path given when matching is always the full S3 object (path), so this did not match.

#### Modification

Change `^` to `/` to match the full filename. Also escape the full stop to look for a literal full stop character.

#### Checklist

_If not applicable, provide explanation of why._

N/A workflow bug

- [ ] Tests updated
- [ ] Docs updated
- [ ] Issue linked in Title

### CODE REVIEW

File: workflows/raster/copy.yaml [hunk 1]
```diff
@@ -3,7 +3,7 @@
 apiVersion: argoproj.io/v1alpha1
 kind: WorkflowTemplate
 metadata:
-  name: copy
+  name: test-copy
```
Comment (l0b0) [inline workflows/raster/copy.yaml:6]: This change looks unrelated. Is it?

Comment (amfage) [reply workflows/raster/copy.yaml:6]: Ah good catch thank you

**Review by l0b0 — APPROVED** [review]

**Review by MDavidson17 — APPROVED** [review]
